src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/custom.py

```diff
@@ -397,83 +693,163 @@ def list_jobs(client, resource_group=None):
     """Output the current content of the file and outputs appended data as the file grows (similar to 'tail -f').
 
-    Press Ctrl-C to interrupt the output.
+    The output will be interrupted as soon as job is completed.
 
     :param BatchAIClient client: the client.
     :param resource_group: name of the resource group.
     :param job_name: job's name.
-    :param directory: job's output directory id.
+    :param output_directory_id: job's output directory id.
+    :param path: path to the file.
     :param file_name: name of the file.
     """
     signal.signal(signal.SIGINT, sigint_handler)
     url = None
     # Wait until the file become available.
+    reported_absence_of_file = False
     while url is None:
-        files = list_files(client, resource_group, job_name, directory)
+        files = list_files(client, resource_group, job_name, output_directory_id, path)
         for f in files:
             if f.name == file_name:
                 url = f.download_url
+                logger.warning('File found with URL "%s". Start streaming', url)
                 break
         if url is None:
+            job = client.get(resource_group, job_name)
+            if job.execution_state in [models.ExecutionState.succeeded, models.ExecutionState.failed]:
+                break
+            if not reported_absence_of_file:
+                logger.warning('The file "%s" not found. Waiting for the job to generate it.', file_name)
             time.sleep(1)
+    if url is None:
+        logger.warning('The file "%s" not found for the completed job.', file_name)
+        return
     # Stream the file
     downloaded = 0
     while True:
-        r = requests.get(url, headers={'Range': 'bytes={0}-'.format(downloaded)},
-                         verify=(not should_disable_connection_verify()))
+        r = requests.get(url, headers={'Range': 'bytes={0}-'.format(downloaded)})
         if int(r.status_code / 100) == 2:
             downloaded += len(r.content)
             print(r.content.decode(), end='')
+        job = client.get(resource_group, job_name)
+        if job.execution_state in [models.ExecutionState.succeeded, models.ExecutionState.failed]:
+            break
+        time.sleep(1)
 
 
-def create_file_server(client, resource_group, file_server_name, json_file=None, vm_size=None, location=None,
-                       user_name=None, ssh_key=None, password=None, disk_count=None, disk_size=None,
-                       storage_sku='Premium_LRS', no_wait=False):
+def wait_for_job_completion(client, resource_group, job_name, check_interval_sec=15):
+    """Waits for specified job completion and setups the exit code to the code of the job.
+
+    :param azure.mgmt.batchai.BatchAIManagementClient client: Batch AI client.
+    :param str resource_group: name of the resource group.
+    :param str job_name: name of the job.
+    :param int check_interval_sec: how often to check the job state.
+    """
```

Comment:
Help text should not added via docstrings on custom commands. These should be mirated to the `help.py` file or the `params.py` file

Reply:
we are not adding any help here. all command and parameters are documented in params.py

Reply:
Per our meeting, please just remove these docstrings.

Reply:
done